src/lib/routes/admin-api/environments.ts

```diff
@@ -73,8 +80,10 @@ export class EnvironmentsController extends Controller {
                 openApiService.validPath({
                     tags: ['Environments'],
                     operationId: 'getEnvironment',
+                    description: 'Gets the environment with `name`',
```

Comment:
This is the environment ID, right? Or is it its name? We should also have a summary.

```suggestion
                    summary: 'Get environment by ID',
                    description: 'Gets the environment whose ID is `name`',
```